components/info/component.go

```diff
@@ -64,7 +67,7 @@ func (c *component) States(ctx context.Context) ([]components.State, error) {
 		},
 		{
 			Name:      StateNameAnnotations,
-			Healthy:   true,
+			Healthy:   err == nil,
```

Comment:
Should we have a separate state? since not all gpud has this file?

Reply:
this is just for manually creating unhealthy state, will be removed once pr is ready for review